tests/unit/s2n_handshake_test.c

```diff
@@ -223,12 +223,12 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_free(client_config));
     }
 
-    /* Stop here if RSA_PSS unsupported */
+    /* Stop here if RSA PSS signing is unsupported */
     if (!s2n_is_rsa_pss_supported()) {
```

Comment:
Since each of these checks applies to one test, can we wrap the individual tests in these "is_supported" checks instead, rather than calling "END_TEST" at multiple places in the file? We're already wrapping them in { } :)